docs/docs/models-definition.md

```diff
@@ -1,25 +1,30 @@
 ## Definition
 
-To define mappings between a model and a table, use the `define` method. Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
+To define mappings between a model and a table in v4, you define a class that inherits from [`Sequelize.Model`](../api/model) and call [`Model.init(attributes, options)`](../api/model#init). The model name is determined by the class name. You need to pass the sequelize instance as `options.sequelize` (see example)
+Sequelize will then automatically add the attributes `createdAt` and `updatedAt` to it. So you will be able to know when the database entry went into the db and when it was updated the last time. If you do not want timestamps on your models, only want some timestamps, or you are working with an existing database where the columns are named something else, jump straight on to [configuration ][0]to see how to do that.
 
+Defining a model with [`sequelize.define()`](../api/sequelize#define) as in previous versions is still supported. Check the v3 docs for examples.
 
 ```js
-var Project = sequelize.define('project', {
+class Project extends Sequelize.Model { }
+Project.init({
   title: Sequelize.STRING,
   description: Sequelize.TEXT
-})
+}, { sequelize })
```

Comment:
So basically something like `static options = {...}; static attributes = {...}`? We still need a `.init()` call though because there is a lot of normalization happening. And I would find it weird when I as a user declare `attributes`, but after the `init()` call they have a completely different structure because everything was normalized. I'm +1 for making `options` and `attributes` public (they are not underscore-prefixed anyway in current versions) but as long as `options` gets a somehow different value after init I prefer passing it to the `init()` function.
The desire to have the attributes and options defined _inside_ the class and not in a function call directly following the class declaration will be solved by decorators.